substratest/factory.py

```diff
@@ -304,7 +304,7 @@ def inc(self):
         return self._idx
 
 
-class _Spec(pydantic.BaseModel, abc.ABC):
+class _Spec(pydantic.BaseModel, abc.ABC):  # noqa: B024
```

Comment:
```suggestion
class _Spec(pydantic.BaseModel):
```

debatable comment :)